### PR TITLE
Review Accordion, Alert, Box and Button pages

### PR DESCRIPTION
**Pull request by raquelarrojo**
Reviewed all examples and added icons examples.

### CODE REVIEW

File: website/screens/components/alert/usage/examples/content.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { DxcAlert, DxcInset, DxcStack } from "@dxc-technology/halstack-react";
+
+const code = `() => {
+  return (
+    <DxcInset space="large">
+      <DxcStack gutter="large">
```
Comment (Lisa159) [inline website/screens/components/alert/usage/examples/content.ts:6]: Maybe add an `align="center"` to center the example?

File: website/screens/components/button/usage/examples/iconUsage.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        <rect fill="none" width="24" height="24" />
+      </g>
+      <g id="Master">
+        <path d="M19,9.3V4h-3v2.6L12,3L2,12h3v8h5v-6h4v6h5v-8h3L19,9.3z M10,10c0-1.1,0.9-2,2-2s2,0.9,2,2H10z" />
+      </g>
+    </svg>
+  );
+
+  return (
+    <DxcInset space="medium">
```
Comment (Lisa159) [inline website/screens/components/button/usage/examples/iconUsage.ts:30]: Maybe to change space to large to have all examples with the same value.

Comment (raquelarrojo) [reply website/screens/components/button/usage/examples/iconUsage.ts:30]: Done!

File: website/screens/components/alert/usage/examples/content.ts (hunk 1 above)
Comment (raquelarrojo) [reply website/screens/components/alert/usage/examples/content.ts:6]: Added `align=center`.

File: website/screens/components/alert/code/examples/modal.ts [hunk 3]
```diff
@@ -14,21 +14,15 @@ const code = `() => {
 
   return (
     <DxcInset space="large">
-      <DxcStack gutter="large" align="center">
-        <DxcButton
-          label="Overlay Alert"
-          onClick={handleVisibility}
-          size="large"
+      <DxcButton label="Show information" onClick={handleVisibility} size="large" />
```
Comment (Lisa159) [inline website/screens/components/alert/code/examples/modal.ts:17]: Maybe remove this size?

**Review by Lisa159 — APPROVED** [review]

**Pull request merged**